# This patch removes the dependency on bkcharts which is not required
# and conflicts with the bokeh dependency.
# See https://github.com/ICB-DCM/pyABC/issues/326
# This patch is not required for 0.10.5 onwards.
diff -Nru pyABC-0.10.4.orig/setup.py pyABC-0.10.4/setup.py
--- pyABC-0.10.4.orig/setup.py	2021-03-06 07:20:12.384045000 +0000
+++ pyABC-0.10.4/setup.py	2021-03-06 07:20:53.224364000 +0000
@@ -20,7 +20,7 @@
                       "dill>=0.2.8.2", "gitpython>=2.1.11",
                       "scikit-learn>=0.21.2", "matplotlib>=3.0.0",
                       "sqlalchemy>=1.3.0", "click>=7.0",
-                      "feather-format>=0.4.0", "bkcharts>=0.2",
+                      "feather-format>=0.4.0",
                       "distributed>=1.23.3", "pygments>=2.2.0",
                       "IPython>=7.0.1", "pyarrow>=0.14.1"],
     extras_require={"R": ["rpy2>=3.2.0", "cffi>=1.13.1"],
